src/go/rdctl/pkg/factoryreset/delete_data_linux.go

```diff
@@ -42,5 +40,7 @@ func DeleteData(paths paths.Paths, removeKubernetesCache bool) error {
 	} else {
 		pathList = append(pathList, filepath.Join(paths.Cache, "updater-longhorn.json"))
 	}
+	appHomeDirs := addAppHomeWithoutSnapshots(filepath.Dir(paths.Lima))
```

Comment:
`appHomeDirs` should have a different name here, since the returned directories aren't in `Paths.AppHome` for Linux. They are in "data home", which we don't have an entry in `Paths` for.

IMO `Paths.AppHome` should point to the value of `XDG_DATA_HOME`, defaulting to `~/.local/share/rancher-desktop`. But that is a separate issue.